Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM locustio/locust:2.17.0
+
+WORKDIR /app    
+COPY . .
+CMD ["--spawn-rate", "1","--run-time", "10s"]
```

Comment:
@gidsg Does line 5 override what is set in locust.conf?

Reply:
These arguments are appended to the locust runtime command, in a similar way to we do currently from Github Actions: https://github.com/communitiesuk/funding-service-design-workflows/blob/a8983426514f3a46eca4e71a9eab49874826e7d1/.github/workflows/run-shared-tests.yml#L152